AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -4754,3 +4754,5 @@ jianshu.com#$#.download-app-guidance { display: none !important; }
 jianshu.com#$#body { overflow: auto !important; }
 jianshu.com##.dt-open-bg[data-scene="leftBtn"]
 jianshu.com##.note-comment-above-ad-wrap
+! https://github.com/AdguardTeam/AdguardFilters/issues/93843
+missouribotanicalgarden.org##div[id^="dnn_ctr13400"]
```

Comment:
It will break popup from this link. I'll fix it directly.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/132207558-1643e689-b7f8-4346-9b13-1d77ec1a7d43.png)

</details><br/>